rpcserver.go

```diff
@@ -502,6 +502,10 @@ type rpcServer struct {
 
 	// selfNode is our own pubkey.
 	selfNode route.Vertex
+
+	// rpcNetParams is the set of currently active parameters for the
+	// rpcServer.
+	rpcNetParams bitcoinNetParams
```

Comment:
We have the full `*Config` available in `cfg`. Can just use that.

Reply:
Yeah but eventually rpcserver will have its own config, guess it doesn't matter one way or the other at this stage.

Reply:
Agree, will do the code move in a later PR